12-master/Dockerfile

```diff
@@ -0,0 +1,167 @@
+FROM postgres:12
+LABEL maintainer="Mike Dillon <mike@appropriate.io>"
```

Comment:
The maintainer label should be synced with the other Dockerfile templates to refer to the PostGIS Project

Reply:
Sorry, I forget I started writing this master template from last year.
I fixes.